content/lactf_2024/flaglang.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Flaglang
+date: 2024-02-18T20:04:40+02:00
+description: Writeup for Flaglang [Lactf 2024]
+author: H0N3YP0T
+tags:
+- web
+draft: false
+---
+___
+
+## Challenge Description
+
+Do you speak the language of the flags?
+
+## Intuition
+
+The website has two dropdown where I can choose two countries and see how they say "Hello world". But, in the list there is also another country
+which is the `Flagistan`. If I try to select it, the website return an error. To resume I need to access to the Flagistan language.
```

Comment:
``return`` --> ``returns``
``I need to access to the`` --> ``I need access to the`` or ``I need to access the``